System.IO.Abstractions/System.IO.Abstractions.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
     <PropertyGroup>
-        <TargetFrameworks>netstandard1.4;netstandard2.0;netcoreapp2.0;netstandard2.1</TargetFrameworks>
+        <TargetFrameworks>netstandard1.4;netcoreapp2.0;netcoreapp2.1;netstandard2.0;netstandard2.1</TargetFrameworks>
```

Comment:
What's the reason for adding a .NET Core 2.1 target? The newly added API is available in .NET Standard 2.1.

Reply:
Because the unit test project is targetting .net core 2.0 and you can't add unit tests for this otherwise.

Reply:
It should be enough to add a .NET Core 2.1 target for the tests and keep this as is.

Reply:
It won't work. I tried that initially to minimize the amount of changes. I can either remove the unit tests and not have tests around those methods or add net core 2.1 as targets for the main library.